nautobot/utilities/tests/test_forms.py

```diff
@@ -429,3 +433,54 @@ def test_invalid_input(self):
         for test in tests:
             with self.assertRaises(forms.ValidationError):
                 self.field.clean(test)
+
+
+class AddressFieldMixinTest(TestCase):
+    """Test cases for the AddressFieldMixin."""
+
+    def setUp(self):
+        """Setting up shared variables for the AddressFieldMixin."""
+        self.ip = IPAddress.objects.create(address="10.0.0.1/24")
+        self.initial = {"address": self.ip}
+
+    def test_address_initial(self):
+        """Ensure initial kwargs for address is passed in."""
+        with mock.patch("nautobot.utilities.forms.forms.forms.ModelForm.__init__") as mock_init:
+            AddressFieldMixin(initial=self.initial, instance={})
+            mock_init.assert_called_with(initial=self.initial, instance=mock.ANY)
```

Comment:
Based on some quick investigation, it looks like in this case in practice, we'd be passed a partially constructed `IPAddress` as `instance`:

```suggestion
            ip_none = IPAddress()
            AddressFieldMixin(initial=self.initial, instance=ip_none)
            mock_init.assert_called_with(initial=self.initial, instance=ip_none)
```